src/hooks/useSyncBrandingSetting.ts

```diff
@@ -0,0 +1,14 @@
+import { atom } from 'jotai'
+import { useUpdateAtom } from 'jotai/utils'
+import { useEffect } from 'react'
+
+export const disableBrandingAtom = atom<boolean>(false)
```

Comment:
If you expect more branding settings, you should shape the atom for them:

    const initialBrandingSettings: BrandingSettings = {}
    const brandingSettingsAtom = atom(initialBrandingSettings)

This will also let you simplify your sync hook, so that any future additions to the `BrandingSettings` type can be effected by just updating the type, and not touching the rest of the implementation.

    export default function useSyncBrandingSettings(settings: BrandingSettings): void {
      const updateSettings = useUpdateAtom(brandingSettingsAtom)
      useEffect(() => updateSettings(settings), [settings, updateSettings])
    }

Reply:
I don't expect more. Do you?